app/models/post.rb

```diff
@@ -1,3 +1,108 @@
+require 'rest-client'
+
 class Post < ApplicationRecord
-  belongs_to :user
+  belongs_to :user  
```

Comment:
turn 
https://firebasestorage.googleapis.com/v0/b/entheogen-a76f2.appspot.com/o/eeg%2F2268f66c-336f-4521-ba48-aa196a2787c0.json?alt=media&token=b209fa87-16a5-44b0-9ea1-4cc16f745aec
to 
https://firebasestorage.googleapis.com/v0/b/entheogen-a76f2.appspot.com/o/eeg%2F1d4df6b8-90ea-4437-8bea-fddb5b2802ba.json?alt=media&token=09a53f76-5b1f-450c-bf3c-dae19fe321d7

Reply:
documentation for streaming, not necessarily helpful for the format of the input json file but helpful in deciphering the fields:

https://web.archive.org/web/20181217055138/http://developer.choosemuse.com/tools/windows-tools/available-data-muse-direct#Muse_Direct_Available_Data 

https://web.archive.org/web/20181204172224/http://developer.choosemuse.com/tools/available-data

https://web.archive.org/web/20181204172224/http://developer.choosemuse.com/tools/available-data

https://web.archive.org/web/20181122004058/http://developer.choosemuse.com/tools/mac-tools/muselab

Reply:
https://urish.medium.com/reactive-brain-waves-af07864bb7d4

Reply:
https://medium.com/@alessandravertrees/physical-computing-final-project-thesis-concept-44d7bd9ccf07

Reply:
https://medium.com/@alessandravertrees/honors-research-paper-ee0002903569

Reply:
https://community.choosemuse.com/categories/challenges